definitions/json/CaseEventToFields/CaseEventToFields.json

```diff
@@ -7819,72 +7819,6 @@
     "ShowSummaryChangeOption": "Y",
     "PageColumnNumber": 1
   },
-  {
```

Comment:
Are these not used in production currently?

Reply:
They are. I've created a CaseEventToFields-Multiple-prod file and moved them to there so prod is unaffected.

Reply:
My bad, I read them both as non prod